doc/harmony-smp_installation_guide.md

```diff
@@ -291,4 +294,35 @@ Example assuming proxy IP address is `192.168.1.1`:
 ```
 
 Please note that when registering SMP with SML, the externally visible address and hostname have to be used, i.e., the
-address and hostname of reverse proxy, not address and hostname of SMP.
\ No newline at end of file
+address and hostname of reverse proxy, not address and hostname of SMP.
+
+### 2.12 Log Files
+
+The SMP application log files are located in the `/var/log/harmony-smp/` directory.
+
+## 3 Version Upgrade
```

Comment:
The same comment about the upgrade process applies here as in PR14: https://github.com/nordic-institute/harmony-common/pull/14#discussion_r803442843

`sudo apt upgrade` causes the service to have a 500 internal error.

Reply:
That's a good point. I updated the packaging scripts to stop the `harmony-smp` service before the upgrade and to start it again after the upgrade if the service has been enabled. However, if the service is disabled then the user must restart the service manually.